### ISSUE TITLE
Refactor: :recycle: Submodules load version from parent pom

### ISSUE THREAD

**Issue by philkohl** [NaluKit/nalu#246]
As a contributor I want to change the version solely in the parent pom. The version should propagate to all submodules.

**Comment by vegegoku** [comment]
Shouldnt `mvn versions:set` do the job?!

**Comment by philkohl** [comment]
Good point, I give that a try later. I had the following in my mind: https://maven.apache.org/maven-ci-friendly.html
